src/caret_analyze/record/interface.py

```diff
@@ -201,8 +216,37 @@ def equals(self, other: RecordsInterface) -> bool:
         """
         pass
 
+    @singledispatchmethod
+    def append(self, arg):
+        raise InvalidArgumentError(f'Unknown argument type: {arg}')
+
+    @append.register
+    def __append_record(self, other: RecordInterface) -> None:
+        self._append_record(other)
+
+    @append.register
+    def __append_record_base(self, other: RecordBase) -> None:
+        self._append_record(other)  # type: ignore
```

Comment:
```suggestion
        # RecordBase object is an argument instead of RecordInterface
        self._append_record(other)
```

Reply:
@takam5f2 
mypy recognizes `type: ignore` to ignore lint check.
[communication](https://mypy.readthedocs.io/en/stable/common_issues.html#silencing-linters)